arkouda/index.py

```diff
@@ -327,6 +324,9 @@ def __getitem__(self, key):
             key = key.values
         return MultiIndex([i[key] for i in self.index])
 
+    def __repr__(self):
+        return f"MultiIndex({repr(self.index)})"
+
```

Comment:
We certainly could! it shouldn't be too difficult. The only reason I didn't was because dtype isn't include in the `__repr__` for `MultiIndex` in `pandas`

```python
>>> df = pd.DataFrame({'a': np.arange(10, dtype=np.int64), 'b': np.arange(10, dtype=np.uint64)})
>>> c = df.groupby(['a','b']).count()
>>> c.index
MultiIndex([(0, 0),
            (1, 1),
            (2, 2),
            (3, 3),
            (4, 4),
            (5, 5),
            (6, 6),
            (7, 7),
            (8, 8),
            (9, 9)],
           names=['a', 'b'])
```